packages/e2e-tests/src/cli/apm-publish.test.js

```diff
@@ -1,45 +1,22 @@
 import test from 'ava'
 import fs from 'fs-extra'
-const path = require('path')
-import execa from 'execa'
+import path from 'path'
+import semverRegex from 'semver-regex'
 import { startBackgroundProcess, normalizeOutput } from '../util'
 
 const ARTIFACT_FILE = 'artifact.json'
 const MANIFEST_FILE = 'manifest.json'
 
-const testSandbox = './.tmp/apm'
+const testSandbox = './.tmp'
+const projectName = 'foobar'
 
-test.beforeEach(() => {
-  fs.ensureDirSync(testSandbox)
-})
-
-test.afterEach(() => {
-  fs.removeSync(testSandbox)
-})
-
-test('should publish an aragon app directory successfully', async t => {
+test.skip('should publish an aragon app directory successfully', async t => {
```

Comment:
I've marked it to be skipped because, even though it passes, it makes `aragon run` fail: `Cannot find transaction pathing`.

Not sure if it's possible to resolve this conflict, but perhaps we shouldn't test deploying to the local devchain since `aragon run` is already doing it. Can we try deploying to rinkeby instead?

Reply:
I tried to use Rinkeby but I'm also having problems with `Cannot find transaction pathing` on some node versions. I'll keep the `.skip` and merge it

Reply:
Oh, I see, it's the same issue as with local host:
It cannot find the transaction path when another deploy is happening at the same time.

Any thoughts @sohkai 	?